### PR TITLE
import_modules_from_strings when loading cfg from file

### PR DESCRIPTION
**Pull request by dreamerlin**
Proposal by @innerlee in https://github.com/open-mmlab/mmaction2/pull/212

### CODE REVIEW

File: tests/test_utils/test_config.py [hunk 1]
```diff
@@ -132,7 +132,7 @@ def test_construct():
 
 
 def test_fromfile():
-    for filename in ['a.py', 'a.b.py', 'b.json', 'c.yaml']:
+    for filename in ['a.py', 'a.b.py', 'b.json', 'c.yaml', 'q.py']:
```
Comment (hellock) [inline tests/test_utils/test_config.py:135]: This unit test cannot tell whether the feature is enabled as expected.

**Comment by hellock** [comment]
ping @dreamerlin

**Comment by innerlee** [comment]
This is different from what I have thought.

**Comment by dreamerlin** [comment]
> ping @dreamerlin

Unittest has been improved

File: tests/data/config/r.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import os.path as osp
+
+data_path = osp.join(osp.dirname(__file__), 'test_import_modules.txt')
+with open(data_path, 'w') as f:
+    f.write('test_import_modules')
```
Comment (hellock) [inline tests/data/config/r.py:5]: We can just set an environment variable instead of writing a file.

**Review by hellock — APPROVED** [review]

**Pull request merged**